desktopexporter/internal/store/util.go

```diff
@@ -1,358 +1,365 @@
 package store
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
-	"strconv"
-	"strings"
 	"time"
 
 	"github.com/CtrlSpice/otel-desktop-viewer/desktopexporter/internal/telemetry"
 )
 
-// Helper function to convert Events to DuckDB list of STRUCT string format
-func eventToString(events []telemetry.EventData) string {
-	eventStrings := []string{}
+// Helper function to convert Events to JSON format
+func MarshalEvents(events []telemetry.EventData) string {
```

Comment:
Am I correct in concluding that if we could use the union types in go, then most of this file would go away?

Reply:
Yep! We'd need to massage the struct we pull out a wee bit, but over all most of it would go away